apple/Tests/FerrostarCoreTests/__Snapshots__/FerrostarCoreTests/testValhallaCostingOptionsJSON.1.txt

```diff
@@ -45,11 +45,15 @@
           - lat: 60.534715
           - lng: -149.543469
         - kind: break
+        ▿ properties: Optional<Data>
+          - some: 30 bytes
```

Comment:
These last 2 tests actually parse "real" responses. I'm not stoked about these just being bytes in the fixture, but on the other hand we have plenty of tests on the Rust side so I'm not inclined to put any more time into this unless someone already has a fix for decoding this as UTF-8 for snapshotting.